packages/@uppy/dashboard/src/components/FileList.js

```diff
@@ -1,14 +1,33 @@
 const FileItem = require('./FileItem/index.js')
+const VirtualList = require('./VirtualList')
 const classNames = require('classnames')
 const { h } = require('preact')
 
+function chunks (list, size) {
+  const chunked = []
+  let currentChunk = []
+  list.forEach((item, i) => {
+    if (currentChunk.length < size) {
+      currentChunk.push(item)
+    } else {
+      chunked.push(currentChunk)
+      currentChunk = [item]
+    }
+  })
+  if (currentChunk.length) chunked.push(currentChunk)
+  return chunked
+}
+
 module.exports = (props) => {
   const noFiles = props.totalFileCount === 0
   const dashboardFilesClass = classNames(
     'uppy-Dashboard-files',
     { 'uppy-Dashboard-files--noFiles': noFiles }
   )
 
+  // 190px height + 2 * 5px margin
+  const rowHeight = 200
```

Comment:
Maybe we could add a comment in CSS that the height is tied to this property in JS as well?

Reply:
It actually also needs different heights for different `uppy-size--xyz` classes too, will also add that. 190 + 2*5px is only for `uppy-size--lg`